packages/cli/src/lib/setup/setupMultihost.ts

```diff
@@ -203,43 +228,50 @@ function Multihost(options) {
                 prompt.get(schema, (err, password) => {
                     if (password && password.password) {
                         if (password.password !== password.passwordRepeat) {
-                            callback('Secret phrases are not equal!');
+                            callback(new Error('Secret phrases are not equal!'));
                         } else {
-                            objects.getObject('system.config', (err, obj) => {
-                                config.multihostService.password = tools.encrypt(obj.native.secret, password.password);
-                                showMHState(config, changed, callback);
+                            this.objects.getObject('system.config', (err, obj) => {
+                                // @ts-expect-error external types not perfect?
+                                config.multihostService.password = tools.encrypt(obj!.native.secret, password.password);
```

Comment:
```suggestion
                                config.multihostService.password = tools.encrypt(obj!.native.secret, password.password as string);
```
this is a good example of "trust me TS, I know better"